setup.cfg

```diff
@@ -64,5 +64,21 @@ skip_covered = True
 skip_empty = True
 
 [flake8]
-ignore = F999,E501,E128,E124
-exclude = .git,*/migrations/*,*/static/CACHE/*
+max-line-length=88
+select = C,E,F,W,B,B950
+ignore = E203, E501, W503
+exclude = venv,.tox,.eggs
```

Comment:
It makes sense? flake8 seems to only scan `*.py` files?

Reply:
Which should be the default, right? I will drop this line, since it's only related to local development. On CI we are working on a clean git fs.